client/src/components/LinkTo.js

```diff
@@ -22,6 +22,10 @@ const LinkTo = ({
   style,
   ...componentProps
 }) => {
+  if (componentProps.previewId) {
+    // Previewed version consumes the previewId, so remove from this one
+    delete componentProps.previewId
+  }
```

Comment:
Since we're not interested in the return value of `delete`, might as well remove the `if (…)` and call `delete` unconditionally.